### DISCUSSION TITLE
Giving single lag to CCF function

### DISCUSSION THREAD

**Discussion by k-fishwick** [pastas/pastas#376] [Q&A]
Hi, we are trying to utilise this function for time series analysis on pulsars. We are running into an issue while trying to calculate errors. we are randomly scrambling an array, inputting this into the function with an array with 1 value of zero and iterating to generate a series of ccf values at zero lag. however when we do this we find the values cycle between 3 distinct values. we have found inputting a list instead of an numpy array works in one case but not another. Any solutions?

**Comment by dbrakenhoff** [comment]
Did you ever manage to fix this? Could you add an example so we can test it ourselves?
